components/card/card.css

```diff
@@ -0,0 +1,215 @@
+.card{
+    display: flex;
+    flex-direction: column;
+    width: 300px;
+    height: max-content;
+    border-radius: 4px;
+    background: var(--white);
+    border: 1px solid var(--off-grey);
+    position: relative;
+}
+
+.text-only-card{
+    padding: 1rem;
+}
+
+.text-only-card .card-title{
+    font-size: 1.5rem;
+    letter-spacing: -0.04em;
+    font-weight: 600;
+}
+
+.text-only-card .card-subtitle{
+    font-size: 0.9rem;
+    font-weight: 350;
+    padding-top: 4px;
+}
+
+.text-only-card .card-text{
+    font-size: 0.875rem;
+    padding-top: 8px;
+    line-height: 1.5;
+
+}
+
+
+.card-vertical .card-image img{
+    padding: 5px;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.card-floating-icon{
+    background: transparent;
+    border: 0;
+    box-shadow: none;
+    position: absolute;
+    font-size: 2rem;
+    width: fit-content;
+    top: 5px;
+    right: 10px;
+    cursor: pointer;
+    
+}
+
+.card-body{
+    padding: 0.8rem;
+}
+
+.card-vertical .card-title{
+    font-size: 1.5rem;
+    letter-spacing: -0.04em;
+    font-weight: 600;
+}
+
+.card-vertical .card-subtitle{
+    font-size: 0.9rem;
+    font-weight: 350;
+    padding-top: 4px;
+}
+
+.card-vertical .card-price{
+    padding-top: 8px;
+    line-height: 1.5;
+}
+
+.price-after-discount{
+    font-size: 1.2rem;
+    font-weight: 550;
+}
+
+.price-before-discount{
+    font-size: 0.9rem;
+    text-decoration: line-through;
+
+}
+
+.discount{
+    padding-left: 5px;
+    color: red;
+}
+
+.btn-box{
+    padding: 0.875rem;
+    margin-top: -1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
u will be using  display: flex;
    justify-content: center;
    align-items: center; very frequently better make a utility class for it